astroquery/mast/core.py

```diff
@@ -777,6 +757,8 @@ def _build_filter_set(self, **filters):
             colType = "discrete"
             if colInfo.get("vot.datatype", colInfo.get("type")) in ("double", "float"):
                 colType = "continuous"
+            elif colInfo.get("treatNumeric"):
```

Comment:
since this has the same outcome as the above if, should this just be `if (condition1) or (condition2): coltype=continuous`?